apps/nextjs-website/src/components/atoms/WebinarHeaderBanner/WebinarHeaderBanner.tsx

```diff
@@ -15,12 +15,20 @@ import { Webinar } from '@/lib/types/webinar';
 import { useTranslations } from 'next-intl';
 
 export type WebinarHeaderBannerProps = {
-  webinars: readonly Webinar[];
+  webinars: Webinar[];
 };
 
 const WebinarHeaderBanner: FC<WebinarHeaderBannerProps> = ({ webinars }) => {
+  // eslint-disable-next-line functional/immutable-data
+  webinars.sort((a, b) => {
+    return (
+      new Date(a.endDateTime ?? 0).getTime() -
+      new Date(b.endDateTime ?? 0).getTime()
```

Comment:
Why did you use `endDateTime`? IMHO `startDateTime` might be more appropriate to cover cases where a webinar starts before and ends after another webinar.
I know this won't happen :)